templates/shortcode-form-grid.php

```diff
@@ -74,388 +74,371 @@ static function($term)use($tag_text_color,$tag_bg_color){
 
-                    echo "
+
+            echo "
                         <div class='give-form-grid-media'>
                             <div class='give-card__media'> $image </div>
 
                             {$renderTags('give-form-grid-media__tags')}
                         </div>
                     ";
-                } elseif(
-                    give_is_setting_enabled($give_settings['form_featured_img'])
-                    && ($imageSrc = $formTemplate->getFormFeaturedImage($form_id))
-                    && $atts['show_featured_image']
-                    && $atts['columns'] === '1')
-                    {
-                        echo "
+        } elseif(
+            give_is_setting_enabled($give_settings['form_featured_img'])
+            && ($imageSrc = $formTemplate->getFormFeaturedImage($form_id))
+            && $atts['show_featured_image']
+            && $atts['columns'] === '1')
+        {
+            echo "
                             <div id='row-media' class='give-form-grid-media'>
                                 <img class='give-form-grid-media' src='$imageSrc' alt='' />
 
                                 {$renderTags('give-form-grid-media__tags')}
                             </div>
                         ";
-                    }
-                ?>
+        }
+        ?>
 
-            <div class="give-form-grid-container">
-                <div class="give-form-grid-content">
-                    <?php
-                    if( !$atts['show_featured_image']){
-                            echo "
+        <div class="give-form-grid-container">
+            <div class="give-form-grid-content">
+                <?php
+                if( !$atts['show_featured_image']){
+                    echo "
                                  <div class='give-form-grid-media' >
                                         {$renderTags('give-form-grid-media__tags_no_image', false)}
                                    </div>
                             ";
-                        }
-                    ?>
+                }
+                ?>
 
-                    <?php
+                <?php
+
+                // Maybe display the form title.
+                if ( true === $atts['show_title'] ) {
+                    printf(
+                        '<h3 class="give-form-grid-content__title">%1$s</h3>',
+                        $formTemplate->getFormHeading( $form_id )
+                    );
+                }
 
-                    // Maybe display the form title.
-                    if ( true === $atts['show_title'] ) {
-                        printf(
-                            '<h3 class="give-form-grid-content__title">%1$s</h3>',
-                            $formTemplate->getFormHeading( $form_id )
+                // Maybe display the form excerpt.
+                if ( true === $atts['show_excerpt'] ) {
+                    if ( $raw_content = $formTemplate->getFormExcerpt( $form_id ) ) {
```

Comment:
Is this a related change? Or rather, is this intended to be a change in this PR?

Reply:
@kjohnson No changes were made it was a formatting difference.
 Heres a photo of the files current state you can see both functions remain in the same order <img width="1139" alt="Screen Shot 2022-10-20 at 9 43 03 AM" src="https://user-images.githubusercontent.com/75056371/197009510-70cb9dfb-468d-4d29-8bf1-c832d58c978e.png">